data/json/proficiencies/misc.json

```diff
@@ -88,7 +88,8 @@
     "can_learn": true,
     "default_time_multiplier": 2,
     "default_fail_multiplier": 1.2,
-    "time_to_learn": "5 h"
+    "time_to_learn": "5 h",
+    "required_proficiencies": [ "prof_pottery" ]
```

Comment:
I left this out on purpose. Pottery represents a different skill from glazing. You can learn one without the other

Reply:
I am dubious, but OK.